turtle.js

```diff
@@ -448,10 +432,10 @@ const _main = function() {
     };
 
     /**@type {HTMLInputElement}*/
-    const cmdBox = doc.getElementById('command');
+    const cmdBox = _doc.getElementById('command');
 
     // Moves up and down in command history
-    cmdBox.addEventListener("keydown", function(e) {
+    cmdBox.addEventListener("keydown", e => {
```

Comment:
We can get rid of both `if ()` blocks by re-writing the whole thing like this:
```js
cmdBox.addEventListener(
  "keydown",
  e => e.key == "ArrowUp" && (cmdIdx = Math.max(cmdIdx - 1, 0)),
  false
);
```

Reply:
We can't assume that if `e.key != 'ArrowUp'` then `e.key == 'ArrowDown'` is `true`. However, the `Math.max` is definitely a good idea! Thank you, I didn't thought of that

Reply:
Oops! When I was navigating your diff. changes I didn't click the plus `+` sign to see more context of it.

So I didn't catch neither the `e.key == 'ArrowDown'` nor the `cmdBox.value = cmddHist[cmdIdx] || '';`!

Still, the current implementation got so many `if ()` blocks that they warrant some simplification.

This is what I've come up w/ in order to shorten the logic of this complex callback:
```js
cmdBox.addEventListener(
  'keydown',
  ({ key }) => {
    if (key == 'ArrowDown')     cmdIdx = Math.min(cmdIdx + 1, cmddHist.length);
    else if (key == 'ArrowUp')  cmdIdx = Math.max(cmdIdx - 1, 0);

    if (key == 'ArrowDown' || key == 'ArrowUp')
      cmdBox.value = cmddHist[cmdIdx] || '';
  },
  false
);
```

Reply:
I missed that comment because GH didn't show me the "resolved conversation" lol. Now I've read it